Jenkinsfile

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env groovy
```

Comment:
I'm not sure why Dustin asked for a PR. Once you fill in the steps in the Jenkinsfile, you should be all set.